static/src/javascripts/projects/common/modules/onward/popular.js

```diff
@@ -40,9 +40,7 @@ define([
     };
 
     MostPopular.prototype.mobileMaximumSlotsReached = function () {
-        return (detect.getBreakpoint() === 'mobile'
-            && config.switches.noMobileTopAd
-            && $('.ad-slot--inline').length > 1) ? true : false;
+        return (detect.getBreakpoint() === 'mobile' && $('.ad-slot--inline').length > 1) ? true : false;
```

Comment:
Isn't this the same as `return detect.getBreakpoint() === 'mobile' && $('.ad-slot--inline').length > 1`?